client/my-sites/customer-home/cards/features/site-preview/site-preview-ellipsis-menu.tsx

```diff
@@ -0,0 +1,64 @@
+import { Gridicon } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import classNames from 'classnames';
+import { useRef, useState } from 'react';
+import PopoverMenu from 'calypso/components/popover-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+export const SitePreviewEllipsisMenu = () => {
+	const { __ } = useI18n();
+
+	const moreButtonRef = useRef( null );
+	const [ isShowingPopover, setIsShowingPopover ] = useState( false );
+
+	const classes = classNames( 'theme__more-button', { 'is-open': isShowingPopover } );
+	const selectedSite = useSelector( getSelectedSite );
+
+	return (
+		<span className={ classes }>
+			<button
+				aria-label={ `More options for site ${ selectedSite?.name }` }
```

Comment:
Should this information not be translated?
```suggestion
				aria-label={ sprintf( __( 'More options for site %s' ), selectedSite?.name ) } // I think we'll need to pull `sprintf` in from `@wordpress/i18n` directly, as it's not exposed via `useI18n()`
```